### DISCUSSION TITLE
Making expo router work on react native for macOS

### DISCUSSION THREAD

**Discussion by Jonatthu** [expo/router#681] [Q&A]
The question is simple:

How to make react native mac os to like expo router?
When I run the macos project, it seems to not recognize the expo router entry import.
Is there any way to modify the index.ts to make it work?

Currently is a simple
```
import "expo-router/entry";
```

**Comment by zRelux** [comment]
Have you tried with `Expo root`?
```
const ctx = require.context('./src/app');
return <ExpoRoot context={ctx} />
```
